### PR TITLE
Adding Cookie3 to tooling folder

### PR DESCRIPTION
**Pull request by a-boruszak**
As a new member of Linea ecosystem, we'd like to add Cookie3 docs into tooling folder.

### CODE REVIEW

File: docs/build-on-linea/tooling/analytics/_category_.json
Comment (emilyJLin95) [inline docs/build-on-linea/tooling/analytics/_category_.json:1]: Is this file supposed to be empty?

Or
```
{
  "label": "Analytics"
}
``

Comment (emilyJLin95) [inline docs/build-on-linea/tooling/analytics/_category_.json:1]: Is this file supposed to be empty?

Or
```
{
  "label": "Analytics"
}
``

Comment (emilyJLin95) [inline docs/build-on-linea/tooling/analytics/_category_.json:1]: Is this file supposed to be empty?

Or
```
{
  "label": "Analytics"
}
``

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]
Left one comment!

File: docs/build-on-linea/tooling/analytics/cookie3.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Cookie3
+---
+
+# Attribution & User Analytics for Web3 Marketers
+
+## What’s Cookie3?
+
+[Cookie3](https://www.cookie3.co/) (aka Google Analytics for Web3) allows you to analyze every visitor and wallet on your website, fuse web2 and web3 with powerful campaigns and full customer journey.
+
+##Benefits of using Cookie3
```
Comment (emilyJLin95) [inline docs/build-on-linea/tooling/analytics/cookie3.md:11]: does this need a space here?

Comment (a-boruszak) [reply docs/build-on-linea/tooling/analytics/cookie3.md:11]: Do you mean space between # and line 3?

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**